sample/src/main/java/com/google/android/material/motion/sample/TossableTapActivity.java

```diff
@@ -50,14 +50,14 @@ public void onLayoutChange(
   }
 
   private void runDemo() {
-    ReactiveProperty<PointF> anchor = ReactiveProperty.of(ViewProperties.CENTER.get(target));
+    ReactiveProperty<PointF> anchor = runtime.get(target).center();
 
     Tossable tossable = new Tossable(ViewProperties.CENTER, anchor);
     runtime.addInteraction(tossable, target);
 
     runtime.addInteraction(new SetPositionOnTap(container), tossable.anchor);
 
     runtime.write(
-      tossable.anchor.getStream(), ReactiveProperty.of(destination, ViewProperties.CENTER));
+      tossable.anchor.getStream(), runtime.get(destination).center());
```

Comment:
Can you also change the other instances of `ReactiveProperty.of()` to `runtime.get()`?

You can do this by right clicking ReactiveProperty.**_of_**() and choose "Find Usages"